docs/src/solutions.md

```diff
@@ -133,15 +133,16 @@ end
 [MathOptInterface](https://www.juliaopt.org/MathOptInterface.jl/v0.9.10/) defines a large
 number of model attributes that can be queried. Examples include
 [`MOI.RelativeGap`](https://www.juliaopt.org/MathOptInterface.jl/v0.9.10/apireference/#MathOptInterface.RelativeGap) and
-[`MOI.SimplexIterations`](https://www.juliaopt.org/MathOptInterface.jl/v0.9.10/apireference/#MathOptInterface.SimplexIterations).
+[`MOI.SimplexIterations`](https://www.juliaopt.org/MathOptInterface.jl/v0.9.10/apireference/#MathOptInterface.SimplexIterations). Some attributes can be directly accessed by getter functions like `JuMP.objective_bound()` and `JuMP.relative_gap()`.
```

Comment:
Instead of "`JuMP.realative_gap()`", use "[`relative_gap`](@ref)". This will link to the documentation. Same for `objective_bound`.

Reply:
done